transport/http/constants.go

```diff
@@ -21,6 +21,15 @@
 package http
 
 const (
+	// ApplicationHeaderPrefix is the prefix added to application headers over
+	// the wire.
+	ApplicationHeaderPrefix = "Rpc-Header-"
+
+	// ContextHeaderPrefix is the prefix added to context headers over the wire.
+	ContextHeaderPrefix = "Context-"
```

Comment:
Well, right now it's the only context header prefix. There's no way to change it. Calling it default implies that there's a way to change it.